test/controllers/cuisines_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
```

Comment:
consider using rspec in your project (that what you will use everyday...)

Reply:
did it on a different branch :)